nodejs/week4/spam-detector/index.js

```diff
@@ -0,0 +1,70 @@
+class Email {
+  constructor(subject, body) {
+    this.subject = subject;
+    this.body = body;
+  }
+}
+
+const spamWords = ["viagra", "offer", "free", "business proposal"];
+
+class SpamDetector {
+  isSpam(email) {
+    const subject = email.subject;
+    const body = email.body;
+    const string = subject + body;
+
+    //more than 60% uppercase characters

```

Comment:
As I understand from the task, you had to check the whole email on uppercase character. Currently you're checking only the subject line.

Reply:
True :O 
changed